shopmatic.shopmakerapp.com.json

```diff
@@ -0,0 +1,16 @@
+{
+  "providerId": "shopmatic",
+  "providerName": "shopmatic",
+  "serviceId": "shopmakerapp.com",
+  "serviceName": "shopmakerapp.com",
+  "logoUrl": "",
+  "description": "Configures shopmakerapp.com service",
+  "records": [
+    {
+      "type": "CNAME",
+      "host": "www",
+      "pointsTo": "%INPUTDOMAIN%"
+    }
+  ]
+}
+
```

Comment:
See shompatic comment above. I would think this is shopmatic.com.shopmakerapp.json (or myshopmatic.com.shopmakerapp.json).  

providerId = shopmatic.com (or myshopmatic.com)
serviceId = shopmakerapp
file = <providerId>.<serviceId>.json